### DISCUSSION TITLE
Elegant way to have a toggle with left and right descriptions?

### DISCUSSION THREAD

**Discussion by wsw70** [quasarframework/quasar#8967] [General - Components / Directives / etc] [answered]
From what I see in the `q-toggle` [documentation](https://quasar.dev/vue-components/toggle), the label can be either on the left or on the right , but never on both sides.

Is there an elegant way to have labels on both sides?

As a workaround I use the following construction but there may be something built-in I missed

     <div><span>left</span><q-toggle value="something" label=""/><span>right</span></div>

**Comment by metalsadman** [comment, marked as answer]
nothing ootb like that, so what you're doing should be fine.
